application-helm/values.yaml

```diff
@@ -319,6 +319,11 @@ rbac:
   #     - list
   #     - watch
 
+initdb:
+  mysql:
+    auto_db_user_create: false
+    permissions: "SELECT, INSERT, UPDATE, DELETE, CREATE TEMPORARY TABLES, LOCK TABLES, EXECUTE"
```

Comment:
should ALTER also not be added?
sync with devs if we have any permissions missing

Reply:
we dont need ALTER permissions for most of our apps, and if we will need, we will override it from their values files